fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/InitConsistentMaterializationContextHook.java

```diff
@@ -49,10 +46,9 @@ public void initMaterializationContext(CascadesContext cascadesContext) {
     }
 
     protected Set<MTMV> getAvailableMTMVs(Set<TableIf> usedTables, CascadesContext cascadesContext) {
-        List<BaseTableInfo> usedBaseTables =
-                usedTables.stream().map(BaseTableInfo::new).collect(Collectors.toList());
         return Env.getCurrentEnv().getMtmvService().getRelationManager()
-                .getAvailableMTMVs(usedBaseTables, cascadesContext.getConnectContext(),
+                .getAvailableMTMVs(cascadesContext.getStatementContext().getCandidateMTMVs(),
```

Comment:
For name issue, is there different between "available" and "candidate"?
If not, better use same word

Reply:
The number of candidates may be greater than the number of available ones. Candidates only determine the status of the materialized view, while available needs to re evaluate whether the data is consistent.